<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>virtualkeyboard</class>
 <widget class="QWidget" name="virtualkeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="mainWidget">
     <widget class="QWidget" name="page_7">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="2" column="3">
          <widget class="QStackedWidget" name="rightSpacerWidget">
           <widget class="QWidget" name="page_11">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="rightSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_12"/>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="spaceBtn">
           <property name="text">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="enterBtn">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="13" column="3">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="2" column="12">
                  <widget class="QPushButton" name="lp">
                   <property name="text">
                    <string>p</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="3">
                  <widget class="QPushButton" name="lx">
                   <property name="text">
                    <string>x</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="3">
                  <widget class="QPushButton" name="le">
                   <property name="text">
                    <string>e</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="12">
                  <widget class="QPushButton" name="n0">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="4">
                  <widget class="QPushButton" name="lr">
                   <property name="text">
                    <string>r</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="6">
                  <widget class="QPushButton" name="lv">
                   <property name="text">
                    <string>v</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="7">
                  <widget class="QPushButton" name="n6">
                   <property name="text">
                    <string>6</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="n2">
                   <property name="text">
                    <string>2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QPushButton" name="n3">
                   <property name="text">
                    <string>3</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="10">
                  <widget class="QPushButton" name="n9">
                   <property name="text">
                    <string>9</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="n1">
                   <property name="text">
                    <string>1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="4">
                  <widget class="QPushButton" name="lc">
                   <property name="text">
                    <string>c</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="1">
                  <widget class="QPushButton" name="shiftBtn">
                   <property name="text">
                    <string>⇧</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="7">
                  <widget class="QPushButton" name="lb">
                   <property name="text">
                    <string>b</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="9">
                  <widget class="QPushButton" name="n8">
                   <property name="text">
                    <string>8</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="7">
                  <widget class="QPushButton" name="ly">
                   <property name="text">
                    <string>y</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="8">
                  <widget class="QPushButton" name="lu">
                   <property name="text">
                    <string>u</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="6">
                  <widget class="QPushButton" name="n5">
                   <property name="text">
                    <string>5</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QPushButton" name="lw">
                   <property name="text">
                    <string>w</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="6">
                  <widget class="QPushButton" name="lt">
                   <property name="text">
                    <string>t</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="9">
                  <widget class="QPushButton" name="lm">
                   <property name="text">
                    <string>m</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="12">
                  <widget class="QPushButton" name="eraseBtn">
                   <property name="text">
                    <string>⌫</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="8">
                  <widget class="QPushButton" name="n7">
                   <property name="text">
                    <string>7</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="lq">
                   <property name="text">
                    <string>q</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="9">
                  <widget class="QPushButton" name="li">
                   <property name="text">
                    <string>i</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="2">
                  <widget class="QPushButton" name="lz">
                   <property name="text">
                    <string>z</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="10">
                  <widget class="QPushButton" name="spt">
                   <property name="text">
                    <string>.</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="4">
                  <widget class="QPushButton" name="n4">
                   <property name="text">
                    <string>4</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="10">
                  <widget class="QPushButton" name="lo">
                   <property name="text">
                    <string>o</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="8">
                  <widget class="QPushButton" name="ln">
                   <property name="text">
                    <string>n</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="12">
                  <widget class="QPushButton" name="ll">
                   <property name="text">
                    <string>l</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="10">
                  <widget class="QPushButton" name="lk">
                   <property name="text">
                    <string>k</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="9">
                  <widget class="QPushButton" name="lj">
                   <property name="text">
                    <string>j</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="8">
                  <widget class="QPushButton" name="lh">
                   <property name="text">
                    <string>h</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="7">
                  <widget class="QPushButton" name="lg">
                   <property name="text">
                    <string>g</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="6">
                  <widget class="QPushButton" name="lf">
                   <property name="text">
                    <string>f</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="4">
                  <widget class="QPushButton" name="ld">
                   <property name="text">
                    <string>d</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="3">
                  <widget class="QPushButton" name="ls">
                   <property name="text">
                    <string>s</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="2">
                  <widget class="QPushButton" name="la">
                   <property name="text">
                    <string>a</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QPushButton" name="sat">
                   <property name="text">
                    <string>#</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2"/>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QStackedWidget" name="leftSpacerWidget">
           <widget class="QWidget" name="page_9">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="leftSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_10"/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
